certify-sunbird-insurance.properties

```diff
@@ -127,17 +66,18 @@ mosip.certify.key-values={\
                      }},\
                     'display': {{'name': 'Health Insurance', \
                                   'locale': 'en', \
-                                  'logo': {'url': 'https://${mosip.api.public.host}/inji/veridonia-logo.png','alt_text': 'a square logo of a Veridonia'},\
+                                  'logo': {'url': 'https://raw.githubusercontent.com/tw-mosip/file-server/master/StayProtectedInsurance.png','alt_text': 'a square logo of a Veridonia'},\
```

Comment:
we should not have the tw-mosip url, can we host logos from gh-pages ?

Reply:
Reverted logo to have the previously used value